### DISCUSSION TITLE
10214FE4-EC97-F166-B7E8-5376CC2C137A

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#116] [General]
# 10214FE4-EC97-F166-B7E8-5376CC2C137A

This addon adds a new high quality texture to our Moon. Taken directly from Celestia Origin.

https://celestia.mobi/addon?item=10214FE4-EC97-F166-B7E8-5376CC2C137A

**Comment by ta269** [comment]
Cooool mooon
